Dom58/Quick-Credit
# #166133620 :  Unity testing

#### What does this PR do?
This PR is based generally on unity testing and few on endpoints finishes
#### Description of Task to be completed?
The task that has finished is like, unit testing from zero up to 64 coverage, finishing to fix all endpoints and rebuild an application.
#### How should this be manually tested?
You should clone `ch-v2-unit-testing-#166133620` to local machine and run npm install to install all dependencies and then you may run `npm run test`
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
PT: #166133620  #166165167 #166162698
#### Screenshots (if appropriate)
![coverage](https://user-images.githubusercontent.com/33524053/58125208-f5aece00-7c07-11e9-9408-9a426f3dcb95.PNG)
#### Questions:
No question, but i still working on the coverage inorder to make it higher
